src/pynxtools/testing/nexus_conversion.py

```diff
@@ -123,12 +126,12 @@ def check_reproducibility_of_nexus(self):
             assert False, "Log files are different"
         for ind, (gen_l, ref_l) in enumerate(zip(gen_lines, ref_lines)):
             if gen_l != ref_l:
-                # skip version conflicts
-                if gen_l.startswith("DEBUG - value: v") and ref_l.startswith(
-                    "DEBUG - value: v"
-                ):
-                    continue
-                assert False, (
-                    f"Log files are different at line {ind}"
-                    f" generated: {gen_l} \n referenced : {ref_l}"
-                )
+                # skip ignored lines (mainly version conflicts)
+                for ignore_line in IGNORE_LINES:
+                    if gen_l.startswith(ignore_line) and ref_l.startswith(ignore_line):
+                        break
+                else:
+                    assert False, (
+                        f"Log files are different at line {ind}"
+                        f" generated: {gen_l} \n referenced : {ref_l}"
+                    )
```

Comment:
Just as an idea, it would be also cool to have the nomad parsing of the individual test files tested with this `testing` module.

Reply:
That's a nice idea. Let me see if I can quickly come up with something for this PR

Reply:
I added something pretty rudimentary. But it can be improved in the future in another PR